packages/messaging/lib/index.js

```diff
@@ -288,7 +288,8 @@ class FirebaseMessagingModule extends FirebaseModule {
   }
 
   /**
-   * @platform android
+   * Called when a message is received while the app is in the background. An app is considered to be in the background if no active window is displayed.
+   * @param {*} handler called with message payload that contains the notification payload that is represented with firebase.messaging.NotificationPayload and the data payload that contains an arbitrary number of key-value pairs sent by developers through the Send API
```

Comment:
Lifted from here https://rnfirebase.io/reference/messaging#setBackgroundMessageHandler

```suggestion
   * @param handler called with an argument of type messaging.RemoteMessage that must be async and return a Promise
```